AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -56,6 +57,7 @@ - (instancetype)initWithIngestion:(nullable MSAppCenterIngestion *)ingestion {
                               withBlock:^(id<MSChannelDelegate> channelDelegate) {
                                 [channelDelegate channelGroup:self didAddChannelUnit:channel];
                               }];
+    [[MSAuthTokenContext sharedInstance] addDelegate:channel];
```

Comment:
We probably don't want to add this to the oneCollector channels.

Reply:
It should definitely be done separately.